### DISCUSSION TITLE
Virus warning

### DISCUSSION THREAD

**Discussion by jojaspi** [dfinke/ImportExcel#1298] [General]
Using ImportExcel in Powershell 7.2.7. triggers a warning by Bitdefender Total Security on plot.ps1, and it blocks “a malicious resource”. Presumably a false positive, since a lot of people on the Bitdefender community page are complaining about the Heur.BZC.ZFV.Boxter.851.9206CBA3 warning in relation to Microsoft-like stuff…

Plot.ps1 starts with:
[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', '', Justification="False positives")]
Don't know what that statement does exactly, but it doesn't prevent Bitdefender to block it...

The Bitdefender warning:

Suspicious activity blocked
3 minutes ago

Feature:
Antivirus

PowerShell tried to load a malicious resource detected as Heur.BZC.ZFV.Boxter.851.9206CBA3 and was blocked. Your device is safe.

**Comment by dfinke** [comment]
Thanks, first I've heard about it. That attribute I believe was used to keep the PSScriptAnalyzer warnings down.
